constants.xml

```diff
@@ -193,7 +193,7 @@
         <!-- Timeout in seconds for ONLY connection that reach our callback function, 0 means no timeout-->
         <CONNECTION_CALLBACK_TIMEOUT>0</CONNECTION_CALLBACK_TIMEOUT>
         <ENABLE_EVM>false</ENABLE_EVM>
-        <EVM_SERVER_BINARY>/usr/local/bin/evm-ds</EVM_SERVER_BINARY>
+        <EVM_SERVER_BINARY>/home/stephen/versions/latest/Zilliqa/evm-ds/target/debug/evm-ds</EVM_SERVER_BINARY>
```

Comment:
I think having your own copy of constants.xml that you feed into your isolated server runs, and you keep it outside of the workstree, will prevent accidental checkins.

Reply:
Yeah, I do that for my tests, its the ./scripts/build.sh that needs a special one in the root directory that is different to the one checked in, I have raised this with platform many times.